frontend/app/components/Spots/SpotPlayer/components/SpotVideoContainer.tsx

```diff
@@ -170,24 +173,43 @@ function SpotVideoContainer({
     }
   }, [spotPlayerStore.playbackRate]);
 
-  const warnText = isProcessing ? 'You’re viewing the entire recording. The trimmed Spot is on its way.' : 'Your trimmed Spot is ready! Please reload the page.'
+  const reloadPage = () => { window.location.reload();  };
+
+  // const warnText = isProcessing ? 'You’re viewing the entire recording. The trimmed Spot is on its way.' : 'Your trimmed Spot is ready!'
```

Comment:
its better to delete instead of commenting out so we don't drag outdated code with us (which will be confusing for new people reading it)